### PR TITLE
Fix packaging issues when releasing

### PR DESCRIPTION
**Pull request by RobPasMue**
When releasing, problems were encountered:
* No packages had been generated nor uploaded as artifacts
* Upload process failed

This fix should also be taken to branch release/0.1

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -81,6 +81,23 @@ jobs:
...
+
+      - name: Upload Wheelhouse
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ env.PACKAGE_NAME }}-v${{ steps.version.outputs.PYANSYS_TOOLS_REPORT_VERSION }}-wheelhouse-${{ runner.os }}-${{ matrix.python-version }}
+          path: '*.zip'
+          retention-days: 7
 
   build_test:
     name: Build and Unit Testing
```
Comment (jorgepiloto) [inline .github/workflows/ci.yml:103]: Suggesting these lines @RobPasMue:

```suggestion
  docs:
    name: Build and deploy documentation
```

**Review by jorgepiloto — APPROVED** [review]
I like the usage of weelhouse for generating various wheels ❤️ 

Also, I noticed that the version of this package is not defined in the `__init__.py` file. This is not mandatory but a common practice, so users can do:

```
import ansys.tools as pyansys_report
print(pyansys_report.__version__)
```

Anyway, this looks really good @RobPasMue 🚀

**Review by MaxJPRey — APPROVED** [review]

**Comment by RobPasMue** [comment]
> I like the usage of wheelhouse for generating various wheels ❤️
> 
> Also, I noticed that the version of this package is not defined in the `__init__.py` file. This is not mandatory but a common practice, so users can do:
> 
> ```
> import ansys.tools as pyansys_report
> print(pyansys_report.__version__)
> ```
> 
> Anyway, this looks really good @RobPasMue 🚀

Thanks @jorgepiloto for the feedback! Much appreciated. I'll open an issue for this feature, since it is not really a fix (I'd like to keep both things separated).

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -81,6 +81,23 @@ jobs:
       - name: Test import
         run: |
           python -c "import ansys.tools.report as report; report.version();"
```
[the remark this answers is not in the document]
Comment (RobPasMue) [reply .github/workflows/ci.yml:83]: Agree! :) Thanks @MaxJPRey

File: .github/workflows/ci.yml (hunk 1 above)
Comment (RobPasMue) [reply .github/workflows/ci.yml:108]: Anyway, maybe this fix could be taken into account later on. I'll keep it in mind. Thanks @jorgepiloto.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -82,6 +83,27 @@ jobs:
...
 
+      - name: Retrieve version
+        run: |
+          echo "::set-output name=PYANSYS_TOOLS_REPORT_VERSION::$(python -c "import ansys.tools.report as report; print(report.version())")"
+        id: version
```
Comment (germa89) [inline .github/workflows/ci.yml:89]: I like this!

Comment (RobPasMue) [reply .github/workflows/ci.yml:89]: Sure you like it... it's coming from PyMAPDL 😄 thanks guys for finding that out!

**Pull request merged**